src/app/layout.tsx

```diff
@@ -20,7 +24,10 @@ export default function RootLayout({
 }>) {
   return (
     <html lang='en'>
-      <body className={manrope.className}>{children}</body>
+      <body className={manrope.className}>
+        <Header />
+        <div className="">{children}</div>
```

Comment:
Just a nit but we can remove className not in use in a follow up PR, also spacing in line 5 and 6

Reply:
I don't understand.
We use the manrope family as the default font family everywhere in the app

Reply:
Oh I meant the className on line 29

Reply:
there was some spacing issues and unsued classname